eng/Versions.props

```diff
@@ -2,7 +2,7 @@
   <!-- Opt-in arcade features -->
   <PropertyGroup>
     <UsingToolVSSDK>true</UsingToolVSSDK>
-    <MicrosoftVSSDKBuildToolsVersion>16.8.3036</MicrosoftVSSDKBuildToolsVersion>
+    <MicrosoftVSSDKBuildToolsVersion>17.0.38-dev17-g08fcf558</MicrosoftVSSDKBuildToolsVersion>
```

Comment:
You also need MicroBuild.Plugins.SwixBuild 1.1.33
https://github.com/dotnet/winforms-designer/pull/3035

Reply:
Also if you want to build and deploy the solution using dev17, then you'd need vssdk of version 17.0.63-dev17-g3f11f5ab or newer, which contains critical bugs fixes.